packages/react-text/src/stories/Default.stories.tsx

```diff
@@ -18,7 +18,7 @@ export const TextStory = (props: TextProps) => {
   );
 };
 
-TextStory.argTypes = {
+Default.argTypes = {
```

Comment:
This seems like a left-over from times when `argTypes` must have been specified manually for Storybook Controls to work. Can the `argTypes` be removed now?

Reply:
AFAIK the `argTypes` were important when the `react-components` were not yet migrated to new DX with nx. Although, not sure if we need them still, maybe @ecraig12345 can give us more info on this?

Reply:
Yeah, the automatic inference works now, so we don’t need them. I was asking just because in some (edge) cases, `argTypes` can be useful even with the new DX and I didn’t check if this is the case. 

More info here:
https://github.com/microsoft/fluentui/blob/master/rfcs/convergence/authoring-stories.md#2-how-to-use-argtypes

Reply:
For text, it does not seem that any of the edge cases apply, thus I'm removing the `argTypes`. In case an issue arises we can add them back.